### ISSUE TITLE
Feature Request, AREPL with code lenses

### ISSUE THREAD

**Issue by Astrantia** [Almenon/AREPL-vscode#124]
I think Quokka.js is for javascript what Arepl is for python, and I like Quokka's approach more because of convenience. Take a look here https://marketplace.visualstudio.com/items?itemName=WallabyJs.quokka-vscode

**Comment by Almenon** [comment]
Yeah, I know of quokka. I use it, as a matter of fact.  I'm not quite sure what you mean by code lens - I'm guessing you mean inline variable values?  I'm planning to add this (see #3 for inline errors) but have not gotten around to it yet.  I'll put it on a milestone so it gets done.  In the meantime I suggest using [wolf](https://marketplace.visualstudio.com/items?itemName=traBpUkciP.wolf).  You can use it at the same time as arepl too.

**Comment by Astrantia** [comment]
@Almenon Wolf looks pretty good, but I'd like this feature to bundled with Arepl. Thanks!

**Comment by Almenon** [comment]
@Astrantia There's lots of work to be done for this but as a first step I added inline error icons. You can opt into this by settings the AREPL.inlineResults setting to true.  Can you test it out and let me know if it works?
